openapi-cli/src/test/resources/generators/schema/swagger/undocument_ref.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: People
+paths:
+  /person:
+    get:
+      summary: Person with given name.
+      description: Return the first person with the give name.
+      operationId: listPerson
+      parameters:
+        - name: name
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The person with the given name.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Person'
```

Comment:
Should we add a test for $ref validation in the responses as well?

Reply:
yeah , we already added the check parser issue , it covered responses as well https://github.com/ballerina-platform/openapi-tools/blob/master/openapi-cli/src/test/java/io/ballerina/openapi/generators/common/SwaggerParserTests.java , current swaggerparser validate response and the requestbody validations but not component schema validation :)